environments/icds-cas/app-processes.yml

```diff
@@ -151,6 +151,51 @@ celery_processes:
       num_workers: 6
     icds_dashboard_reports_queue:
       concurrency: 2
+  'celery11':
+    ucr_indicator_queue:
+      concurrency: 4
+    reminder_case_update_queue:
+      pooling: gevent
+      concurrency: 5
+      num_workers: 6
+    icds_dashboard_reports_queue:
+      concurrency: 2
+  'celery12':
+    ucr_indicator_queue:
+      concurrency: 4
+    reminder_case_update_queue:
+      pooling: gevent
+      concurrency: 5
+      num_workers: 6
+    icds_dashboard_reports_queue:
+      concurrency: 2
+  'celery13':
+    ucr_indicator_queue:
+      concurrency: 4
+    reminder_case_update_queue:
+      pooling: gevent
+      concurrency: 5
+      num_workers: 6
+    icds_dashboard_reports_queue:
+      concurrency: 2
+  'celery14':
+    ucr_indicator_queue:
+      concurrency: 4
+    reminder_case_update_queue:
+      pooling: gevent
+      concurrency: 5
+      num_workers: 6
+    icds_dashboard_reports_queue:
+      concurrency: 2
+  'celery15':
+    ucr_indicator_queue:
+      concurrency: 4
+    reminder_case_update_queue:
+      pooling: gevent
+      concurrency: 5
+      num_workers: 6
+    icds_dashboard_reports_queue:
```

Comment:
To be honest, I don't really have a system for these. I'm just copying the most common celery config as I don't know of any bottlenecks we have at the moment.